packages/aws-cdk-lib/aws-stepfunctions/lib/state-graph.ts

```diff
@@ -113,6 +123,7 @@ export class StateGraph {
       StartAt: this.startState.stateId,
       States: states,
       TimeoutSeconds: this.timeout && this.timeout.toSeconds(),
+      Comment: this.comment,
```

Comment:
I don't think changes to `StateGraph` are necessary anymore. `StateGraph` is exposed but not meant to be used externally. No one should be setting `comment` on `StateGraph` and internally we are not doing that anymore either. So there's no need to have this prop.

(This means that the ordering between `...graphJson` and `Comment: sfnProps.comment` won't matter anymore, but its fine to have graphJson in front).

Reply:
haha... I forgot i added it to state-graph, totally makes sense to take it out - updated